fundingmanager.go

```diff
@@ -397,14 +388,32 @@ func (f *fundingManager) Start() error {
 			// The funding transaction was confirmed, but we did not
 			// successfully send the fundingLocked message to the
 			// peer, so let's do that now.
-			go f.sendFundingLockedAndAnnounceChannel(channel,
-				shortChanID)
+			f.wg.Add(1)
+			go func() {
+				defer f.wg.Done()
+				f.sendFundingLockedAndAnnounceChannel(channel,
+					shortChanID)
+			}()
 
 		case fundingLockedSent:
 			// fundingLocked was sent to peer, but the channel
 			// announcement was not sent.
-			go f.sendChannelAnnouncement(channel, lnChannel,
-				shortChanID)
+			f.wg.Add(1)
+			go func() {
+				defer f.wg.Done()
+
+				lnChannel, err := lnwallet.NewLightningChannel(
```

Comment:
Passing in `ChainNotifier` instance here isn't necessary as this won't be the version of the channel that's used within the `breachArbiter` or `channelLink` structs. Passing in a `nil` parameter for the `ChainNotifier` argument means that a `closeObsever` won't be launched, and no notifications will be requested.